docusaurus/docs-internals/editor-controls/multivalue-controls/margin.md

```diff
@@ -1,32 +1,47 @@
 | `devices?`      | `"all"` \|   `"desktop"`  \| `"responsive"`  |   `"all"`    | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
-| `disabled?`     | `boolean`                        |       `false`       | Configure the condition under which the control is disabled or enabled.                                                        |
-| `config?.edges` | `"all"` \|      `"vertical"` \|  `"horizontal"`  |   `"all"`    | The edges configuration value determines which axis actions are enabled. When set to horizontal, it enables actions along the x-axis. When set to vertical, it enables actions along the y-axis.                                                                                                                                                                                                                   |
-| `config?.units` | `Array<Unit>`                    |    `["px","%"]`     | The units of measurement displayed next to the slider value.<br/> Accepts CSS units (`"px"`, `"%"`) <br/> <br/> `Unit = "px" \| "%"` |
+| `disabled?`     | `boolean`      |       `false`       | Configure the condition under which the control is disabled or enabled.                                                                     |
+| `states?`       | `Array<State>` | [`"normal"`] | Allows for different styles based on the element's state <br/> <br/> <b>`State = "normal" \| "hover" \| "active"`</b> <br/> <br/> `"normal"` - the normal state of an element, <br/> `"hover"` - the state when the element is hovered over, <br/> `"active"` - the state when the element is active (e.g., current page in pagination).                                                                                         |
+| `config?.edges` | `"all"` \|      `"vertical"` \| `"horizontal"`  |   `"all"`    | The edges configuration value determines which axis actions are enabled. When set to horizontal, it enables actions along the x-axis. When set to vertical, it enables actions along the y-axis.                                                                                                                                                                                                                   |
+| `config?.units` | `Array<Unit>`  |    `["px","%"]`     | The units of measurement displayed next to the slider value.<br/> Accepts CSS units (`"px"`, `"%"`) <br/> <br/> `Unit = "px" \| "%"`. |
+| `default?`      | `Default`      |      -       | The default control value. <br/> <br/> <b>`Default: { marginType: "grouped"; margin: 0; marginSuffix: "px"; marginTop: 0; marginTopSuffix: "px"; marginRight: 0; marginRightSuffix: "px"; marginBottom: 0; marginBottomSuffix: "px"; marginLeft: 0; marginLeftSuffix: "px";}`</b> <br/> <br/> |
```

Comment:
about `default`

review one more time your keys, because as i see in `converters`, your control is trying to get another keys instead of yours
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/c37f3bcd-ba56-4c5d-ba92-515da1e1253c)